util/dynamic_bloom.h

```diff
@@ -21,30 +21,26 @@ class Slice;
 class Allocator;
 class Logger;
 
+// A Bloom filter intended only to be used in memory, never serialized in a way
+// that could lead to schema incompatibility. Supports opt-in lock-free
+// concurrent access.
```

Comment:
Can you document the bloom filter format here in the comment?
I know we don't do it now but these days people have feedback that our code base needs to improve on maintainability and now we need to have better requirement on commenting.

It would also help me do the code review:)

Reply:
I put high-level requirements and assumptions in the class comment and implementation details in a comment near the probing logic.